L1Trigger/L1THGCal/test/HGCalTriggerGeomTesterV9Imp3.cc

```diff
@@ -1014,6 +1023,21 @@ void HGCalTriggerGeomTesterV9Imp3::fillTriggerGeometry()
       triggerCellWaferV_ = id_si_trig.waferV();
       triggerCellU_ = id_si_trig.triggerCellU();
       triggerCellV_ = id_si_trig.triggerCellV();
+
+      if (triggercell_cells.second.begin() == triggercell_cells.second.end()) {
+        throw cms::Exception("BadGeometry") << "HGCalTriggerGeometry: No cells!";
```

Comment:
Updated. 
We lose nothing by being pedantic, and we never know what future developments might bring. 
I've moved  the exception to the top, so that it is thrown as soon as possible.